news/4.35/platform.html

```diff
@@ -72,9 +72,19 @@ <h2>Themes and Styling </h2>
   <!-- ******************* General Updates ************************************* -->
   <tr>
     <td id="GeneralUpdates" class="section" colspan="2">
-    <h2>General Updates </h2>
+    <h2>General Updates </h2>    
     </td>
   </tr>
+  <tr id="sysPropAppName"> <!-- https://github.com/eclipse-platform/eclipse.platform.ui/pull/1902 -->
+      <td class="title">New System Property: appName</td>
```

Comment:
I think the new property name is `eclipse.appName` rather than `appName`?

Reply:
New PR can be found here: https://github.com/eclipse-platform/www.eclipse.org-eclipse/pull/261